RadixWallet/Features/CreatePersona/Coordinator/CreatePersonaCoordinator+Reducer.swift

```diff
@@ -84,7 +98,14 @@ public struct CreatePersonaCoordinator: Sendable, FeatureReducer {
 		case completed
 	}
 
+	public enum InternalAction: Sendable, Equatable {
+		case derivePublicKeys
```

Comment:
It is in fact a single key, so maybe `derivePublicKey`

Reply:
changing it here, but shouldn't it be everywhere?

Reply:
Yes indeed